### DISCUSSION TITLE
Is there an issue in firehose causing the change in commit.seq ?

### DISCUSSION THREAD

**Discussion by Li-WeiCheng** [MarshalX/atproto#459] [Questions] [answered]
Hi,

I noticed a significant change in the commit.seq.

For instance, the commit.seq (cursor) stopped increasing and remained at 4170910000 around 2024/11/21 11:00:00 PM GMT+0.
                   Additionally, commit.seq (cursor) appears to have decreased to  274452424 around 2024/11/22 02:00:00 GMT+0.

Could you kindly confirm if this behavior is expected or if there might be an issue?

Thanks :)

**Comment by MarshalX** [comment, marked as answer]
Hi, bsky team has some issue

<img width="1214" alt="image" src="https://github.com/user-attachments/assets/5310781a-d3a2-41ec-8e6d-76f6d0dee18b">
